uint8 robot_id
float64 x_req
float64 y_req
---
float64 x_resp
float64 y_resp